pylivetrader/backend/alpaca.py

```diff
@@ -287,6 +288,27 @@ def orders(self):
             for o in self._api.list_orders('all')
         }
 
+    def all_orders(self, status=None):
+        until = pd.Timestamp.utcnow().isoformat()
+        all_orders = {}
+        batch_size = 500
+        orders = self._api.list_orders(status, batch_size, until=until)
+        while(True):
+            batch_orders = {
+                o.client_order_id: self._order2zp(o)
+                for o in orders
+            }
+            # get the timestamp of the earliest order in the batch
+            # add 1 second to avoid orders being dropped at batch limit
+            until = pd.Timestamp(
+                orders[-1].submitted_at + timedelta(seconds=1)
```

Comment:
I thought you wouldn't need this extra care since it's exclusive. You see the granularity is second?

Reply:
No you're right, was written with the assumption that it was second granularity based on the docs, but since it's got subsecond info it should be fine to remove this